### PR TITLE
Chinese doc: credit card fraud, DSW demo

### PR DESCRIPTION
**Pull request by lhw362950217**
Related to #3014

### CODE REVIEW

**Review by shendiaomo — DISMISSED** [review]

File: doc/sqlflow.org_cn/fraud-dnn.cn.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# 信用卡欺诈检测
```
Comment (typhoonzero) [inline doc/sqlflow.org_cn/fraud-dnn.cn.md]: 应该增加一层 `tutorials` 目录？

File: doc/sqlflow.org_cn/fraud-dnn.cn.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+```bash
+unzip creditcard.csv.zip
+```
+
+#### 建表
+
+我们在SQLFlow command-line上用以下语句建立数据表：
+
+```
+CREATE DATABASE IF NOT EXISTS creditcard;
```
Comment (typhoonzero) [inline doc/sqlflow.org_cn/fraud-dnn.cn.md]: 建表应该也可以在 jupyter notebook 里执行？

File: doc/sqlflow.org_cn/fraud-dnn.cn.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    v24 FLOAT,v25 FLOAT,v26 FLOAT,v27 FLOAT,
+    v28 FLOAT,amount FLOAT, class int);
+```
+
+#### 导入CSV
+
+建表完成后，将creditcard.csv的内容导入数据表。
+
+```
+LOAD DATA LOCAL INFILE 'creditcard.csv'
```
Comment (typhoonzero) [inline doc/sqlflow.org_cn/fraud-dnn.cn.md]: 直接执行这个大概率会不成功。一些 MySQL 部署会设置 `LOAD DATA` 的安全设置：https://dev.mysql.com/doc/refman/8.0/en/load-data-local-security.html ，如果 `creditcard.csv` 在当前目录很可能不会加载成功。可以用 Python 脚本或者说明导入 csv 文件的各项配置。

File: doc/sqlflow.org_cn/fraud-dnn.cn.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+%%sqlflow
+SELECT * FROM creditcard.train limit 780
+TO TRAIN DNNClassifier
+WITH train.batch_size=32,
+    model.n_classes=2,
+    model.batch_norm=True,
+    model.hidden_units=[64, 32],
+    optimizer.learning_rate=0.1,
+    train.epoch=20,
+    validation.select="select * from creditcard.train limit 780"
```
Comment (typhoonzero) [inline doc/sqlflow.org_cn/fraud-dnn.cn.md]: 使用训练数据作为验证数据不是常见的方法，可以在之前切分下训练/预测数据。

File: doc/sqlflow.org_cn/fraud-dnn.cn.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+LABEL class
+INTO creditcard.my_fraud_dnn_model;
+```
+
+我们可以从日志链接中看到训练效果，由于训练集数据具有一定随机性，您所看到的结果可能稍有不同：
+
+```json
+{'accuracy': 0.9794872, 'accuracy_baseline': 0.63076925, 'auc': 0.9988673, 'auc_precision_recall': 0.999346, 'average_loss': 0.07168599, 'label/mean': 0.63076925, 'loss': 0.07210991, 'precision': 0.98373985, 'prediction/mean': 0.6431263, 'recall': 0.98373985, 'global_step': 500}
```
Comment (typhoonzero) [inline doc/sqlflow.org_cn/fraud-dnn.cn.md]: AUC 约等于 1了，其实是过拟合了，这里可能有会有歧义。

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 1 above)
Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: done

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 2 above)
Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: 加载数据这一段是给用户看的，不希望他操作，我们已经把数据实现准备好了。在dsw Notebook里不太好导入数据。

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 3 above)
Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: done

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 4 above)
Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: done

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 5 above)
Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: done

File: doc/sqlflow.org_cn/fraud-dnn.cn.md (hunk 3 above)
Comment (typhoonzero) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: 好像不光是绝对路径的事情。这个路径需要在mysql client中配置貌似。

Comment (lhw362950217) [reply doc/sqlflow.org_cn/fraud-dnn.cn.md]: done，可以通过设置 set global local_infile = 1; 来允许本地上传

**Review by typhoonzero — APPROVED** [review]
